src/components/nimbleAutoComplete/NimbleAutoComplete.tsx

```diff
@@ -42,7 +42,7 @@ export const NimbleAutoComplete: React.FC<NimbleAutoCompleteProps> = ({
   labelSize = 14,
   labelWeight = '600',
   placeholder,
-  fontFamily = 'Roboto,Helvetica,Arial,sans-serif',
+  fontFamily,
```

Comment:
fontFamily is optional prop, so don't remove this default value if prop is optional